lemur/certificates/service.py

```diff
@@ -1274,3 +1274,43 @@ def allowed_issuance_for_domain(common_name, extensions):
     # lemur UI copies CN as SAN (x509.DNSName). Permission check for CN might already be covered above.
     if check_permission_for_cn:
         is_authorized_for_domain(common_name)
+
+
+def send_certificate_expiration_metrics():
+    """
+    Iterate over each certificate and emit a metric for how many days until expiration.
+    """
+    success = failure = 0
+    # get eligible certificates
+    certificates = get_certificates()
```

Comment:
This seems to filter out certificates which have notifications disabled and are not expiring within 30 days. We want this to monitor all non-expired certificates so this behavior is probably not what we want here. So I'd suggest creating another function here which does something similar as  `get_certificates()` but has the required filtering constraints.

Reply:
Understood, so maybe something like,
```
certificates = database.db.session.query(Certificate).filter(
        Certificate.expired == false(),
        Certificate.revoked == false()
    )
```